index.bs

```diff
@@ -180,6 +180,26 @@ by showing how many sites are willing to ask for intrusive and unnecessary permi
 
 <h2 id="api-across-languages">API Design Across Languages</h2>
 
+<h3 id="simplicity">Prefer simple solutions</h3>
+<!-- was "Avoid Needless Complexity" in the HTML Design Principles -->
+
+Simple solutions are preferred to complex ones, when possible.
+Simpler features are easier for user agents to implement and test,
+more likely to be interoperable,
+and easier for authors to understand. [[LEAST-POWER]]
+
+But this should not be used as an excuse to avoid satisfying the other principles.
+
+<h3 id="handle-errors">Prefer graceful error recovery</h3>
+<!-- was "Handle Erros" in the HTML Design Principles -->
+
+Error handling should be defined
+so that interoperable implementations can be achieved.
+
+Prefer graceful error recovery to hard failure,
```

Comment:
I think this needs a lot more nuance as a lot of errors do not surface to the user and those that do are often security-related whereby hard failure is in fact preferred.

https://tools.ietf.org/html/draft-iab-protocol-maintenance is a good read on this too.

Reply:
I think this may require enough additional work that I'm going to drop it from this PR and make a separate PR just for this principle.